### ISSUE TITLE
custom page documentation incorrect for Docker install

### ISSUE THREAD

**Issue by deeztek** [causefx/Organizr#2009]
###### Operating System: Ubuntu/Docker
<hr>

##### Problem Description:
Custom Pages documentation at: https://docs.organizr.app/help/custom-work/custom-pages does not seem to be valid for Docker installs. There is no "pages" directory under the "data" folder.

<hr>

##### Reproduction Steps:

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by tronyx** [comment]
Seems like a simple note needs to be added for you to create it if it doesn't exist. Have you tried that?

**Comment by causefx** [comment]
Yes – I believe the creation of folder is needed if not present.

**Comment by deeztek** [comment]
It looks like the "pages" folder is located under "/config/www/organizr/api/pages/". I was able to create a custom page and it displays, however I'm not able to insert the following between the <script> and </script>. It throws a 500 error:

```
<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>

<button id="zammad-feedback-form">Feedback</button>

<script id="zammad_form_script" src="https://helpdesk.domain.tld/assets/form/form.js"></script>

<script>
$(function() {
  $('#zammad-feedback-form').ZammadForm({
    messageTitle: 'Feedback Form',
    messageSubmit: 'Submit',
    messageThankYou: 'Thank you for your inquiry (#%s)! We\'ll contact you as soon as possible.',
    modal: true
  });
});
</script>
```

It throws a 500 error.

**Comment by causefx** [comment]
The folder inside data – i.e. /data/pages is mapped dynamically to get pulled in.  I can test further if you like.
